src/Manager/Environment/Install.tsx

```diff
@@ -14,34 +19,83 @@ import Button from './Button';
     const anyInProgress = useSelector(isAnyToolchainInProgress);
+    const [showExperimentalDialog, setShowExperimentalDialog] = useState(false);
 
     const { platform } = process;
-    const onClick = (() => {
+    const onClick = () => {
         switch (platform) {
             case 'darwin':
-                return () => dispatch(install(environment, false));
+                return dispatch(install(environment, false));
             case 'linux':
-                return () =>
-                    dispatch(showConfirmInstallDirDialog(version(environment)));
+                return dispatch(
+                    showConfirmInstallDirDialog(version(environment))
+                );
             case 'win32':
-                return () =>
-                    dispatch(showConfirmInstallDirDialog(version(environment)));
+                return dispatch(
+                    showConfirmInstallDirDialog(version(environment))
+                );
         }
-    })();
+    };
 
     return !isOnlyAvailable(environment) ? null : (
-        <Button
-            icon="x-mdi-briefcase-download-outline"
-            onClick={onClick}
-            label="Install"
-            variant="secondary"
-            disabled={anyInProgress}
-        />
+        <>
+            <GenericDialog
+                isVisible={showExperimentalDialog}
+                headerIcon="alert"
+                title="Experimental Toolchain and SDK"
+                footer={
+                    <>
+                        <DialogButton
+                            variant="danger"
+                            onClick={() => {
+                                setShowExperimentalDialog(false);
+                                onClick();
+                            }}
+                        >
+                            Accept the risks and install
+                        </DialogButton>
+                        <DialogButton
+                            onClick={() => {
+                                setShowExperimentalDialog(false);
+                                openUrl(
+                                    'https://github.com/microsoft/vscode-docs/blob/vnext/release-notes/v1_63.md#pre-release-extensions'
+                                );
+                            }}
+                        >
+                            How use VSCode pre-release
+                        </DialogButton>
+                        <DialogButton
+                            onClick={() => setShowExperimentalDialog(false)}
+                        >
+                            Cancel
+                        </DialogButton>
+                    </>
+                }
+            >
+                Support for nRF Connect SDK {environment.version} in nRF Connect
+                for VS Code is experimental. VS Code users that need{' '}
+                {environment.version} should use the preview version of the
+                extension, please refer to VS Code for details.
```

Comment:
Do we really need `{environment.version}` if this only appears next to an SDK version entry?
```suggestion
                Support for this version in nRF Connect for VS Code is experimental. If you need to use this version, use the pre-release version of the extension.
```
To match phrasing from https://github.com/nrfconnect/vscode-nrf-connect/pull/4034